script.js

```diff
@@ -109,18 +109,21 @@ function drawCarouselSlideLeft() {
         step = slider.length - 1;
     }
 };
-function changesButtonsStatusDisable(disable) {
-    if (disable === true) {
-        buttonRight.disabled = true;
-        buttonLeft.disabled = true;
-    }
-    else {
-        buttonRight.disabled = false;
-        buttonLeft.disabled = false;
+
+function throttle (callback, limit) {
```

Comment:
What is it throttling? Is is just making the car stop? Or  can I grab this function and use wherever I want?

Reply:
Wherever I want.
I don't need here debaunce function (count the delay from the last call), i need simple 500ms delay, so I used a simpler one - throttle.